packages/tasks/src/hardware.ts

```diff
@@ -128,6 +128,10 @@ export const SKUS = {
 				tflops: 12.74,
 				memory: [12, 8],
 			},
+			"RTX 3050 Mobile":{
+				tflops: 7.639,
+				memory:[6],
```

Comment:
The first version was slower and had 4 GB: https://www.techpowerup.com/gpu-specs/geforce-rtx-3050-mobile.c3788

Let's keep this one for now (Numbers match this reference: https://www.techpowerup.com/gpu-specs/geforce-rtx-3050-mobile-refresh-6-gb.c3971)